### ISSUE TITLE
cannot find package "github.com/drone/drone-go/plugin"

### ISSUE THREAD

**Issue by asaf** [drone/drone-go#27]
Hey,

I'm trying to build some plugin,

When I run
`go get "github.com/drone/drone-go/plugin"`

I get:

```bash
package github.com/drone/drone-go/plugin: cannot find package "github.com/drone/drone-go/plugin" in any of:
	/usr/local/go/src/github.com/drone/drone-go/plugin (from $GOROOT)
	/Users/someone/data/dev/go/src/github.com/drone/drone-go/plugin (from $GOPATH)
```

(I'm not familiar enough with _Go_ deps, so that might sound silly)

Thanks.

**Comment by asaf** [comment]
This will be fixed as apart of 0.5 effort.

**Comment by tboerger** [comment]
Just as a hint, if you really need to build a plugin for 0.4 you should use vendoring and pin it to eaa41f7836a191224ec5702d2458db07882ec269
